elasticsearch/test/client-nodes-resources.json

```diff
@@ -0,0 +1,101 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
```

Comment:
I guess after thinking about it here, I would have to say yes they are left over from test but intentionally included them.